frontend/src/metabase/query_builder/components/filters/modals/InlineDatePicker/InlineDatePicker.styled.ts

```diff
@@ -6,6 +6,9 @@ import Button from "metabase/core/components/Button";
 
 export const OptionContainer = styled.div`
   font-weight: bold;
+  display: flex;
+  flex-wrap: wrap;
+  gap: ${space(1)};
```

Comment:
we get cleaner spacing between elements with flex gap than using right and bottom margins.